src/TAlias.cpp

```diff
@@ -143,11 +143,6 @@ bool TAlias::match(const QString & toMatch )
         qDebug()<<"CRITICAL ERROR: SHOULD NOT HAPPEN->pcre_info() got wrong num of cap groups ovector only has room for %d captured substrings\n";
     }
 
-    if( rc < 0 )
```

Comment:
Can't add a comment on a line not being edited but that `switch` at line 127 is pointless - it should just be the `goto MUD_ERROR;`that is redundant here...

{Now __that__ is something I don't expect to say too offen, "you just need a goto"! :scream:}